packages/dashboard/src/portal/views/Transfer.vue

```diff
@@ -118,18 +122,30 @@ export default class TransferView extends Vue {
   isTransferValidTwinId = false;
   receptinTwinId = "";
   accountTwinIds: any = [];
-  targetError = "";
+  targetErrorTwin = "";
+  targetErrorAddress = "";
 
   queryClient = new QueryClient(window.configs.APP_API_URL);
   client = new Client({ url: window.configs.APP_API_URL });
 
   async onInputValueChanged() {
     await this.$nextTick();
     this.transferTwinIdCheck();
+    this.targetErrorTwin = "";
   }
 
   async transferTwinIdCheck() {
     await this.$nextTick();
+    if (this.receptinTwinId.length === 0) {
+      this.targetErrorTwin = "This field is required";
+      return;
+    }
```

Comment:
the rest error messages should be handled here too and removed from the form's rules?